README.md

```diff
@@ -23,4 +25,10 @@ Save-Module -Name 'Carbon.Registry' -Path '.'
 Import-Module -Name '.\Carbon.Registry'
 ```
 
-# Commands
+## Commands
+
+* `Get-CRegistryKeyValue`

```

Comment:
Include the function SYNOPSIS or part of its DESCRIPTION when documenting it in the README?